runtime/onert/core/include/ir/train/TrainingInfo.h

```diff
@@ -65,6 +68,7 @@ class TrainingInfo final
   bool isValid() const;
 
 private:
+  uint32_t _version;
```

Comment:
Do you also have a plan to check version compatibility?

Reply:
I think we wiill have to?
But for now, I just added this attribute for copying model metadata.